govcd/vdc.go

```diff
@@ -837,3 +836,41 @@ func (vdc *Vdc) buildNsxvNetworkServiceEndpointURL(optionalSuffix string) (strin
 
 	return hostname, nil
 }
+
+// GetEdgeGatewayReferenceList retrieves a list of edge gateway references.
+// Unlike other entities under Vdc, edge gateway are not listed in the parent entities
```

Comment:
Nit: "Vdc" should be "VDC" in comments (sentences for humans), while "Vdc" should be used in function names.

Reply:
"Fixed". On Vaidotas advice, I remove this function completely, and made a similar private function public.